forge/How to make a new extension 3 3/how-to-make-a-new-extension-3-3.md

```diff
@@ -0,0 +1,285 @@
+<!--
+parent: 'TAO 3 1'
+created_at: '2015-06-18 13:17:42'
+updated_at: '2018-10-10 08:48:53'
+authors:
+    - 'Christophe Garcia'
+    - 'Martin Nicholson'
+tags:
+    - 'Legacy Versions:TAO 3.0'
+-->
+
+# How to make a new extension in TAO 3.3
+
+## Preparation
+
+1. Install TAO (3.x) (https://github.com/oat-sa/package-tao)
+2. Download `taoDevTools` extension using composer:
+    `composer require "oat-sa/extension-tao-devtools" —dev`
+3. Make sure the extension has been installed (See https://hub.taocloud.org/articles/wiki/installing-a-new-extension)
+
```

Comment:
Link this to the article on GitHub instead, use a proper md link
https://github.com/oat-sa/taohub-articles/blob/master/forge/Wiki/installing-a-new-extension.md